doc/user_guide.md

```diff
@@ -33,6 +33,16 @@ Through the use of [coroutines](http://en.wikipedia.org/wiki/Coroutine) this MQT
 * The library can send and receive simultaneously on a single socket.
 * Communication requirements of your application will not interfere with the usability of your device.
 
+### Secure File Transfer (SFT)
```

Comment:
I don't know, just a suggestion: if one searches for Firmware Update then he hardly finds this "flagship" feature. So I'd mention 'Firmware Update (FWU)' expression too somewhere. Maybe in this section title, why isn't it:
'### Secure File Transfer (SFT), Firmware Update (FWU)' ?
Just an idea.